src/components/SideBySide/SideBySide.vue

```diff
@@ -0,0 +1,146 @@
+<template lang="html">
+    <div v-if="!isMobile" class="gallery-side-by-side">
+        <div class="images">
+            <div
+                v-for="(image, i) in images"
+                v-if="images.length"
+                class="wrapper"
+            >
+                <wp-image
+                    v-if="image"
+                    :key="image.id"
+                    :image="image"
+                    :aspect-ratio="130"
+                />
+            </div>
+        </div>
+        <div class="text">
+            <wp-content v-for="(text, i) in texts" :key="text" :html="text" />
+        </div>
+    </div>
+    <div v-else class="gallery-side-by-side">
```

Comment:
My top level note says I think we want this on page always, so you can remove the IF but otherwise I'd have said this:

> 
> Change the order of this if, it's a little more user friendly to have the positive statement first, so do this:
> ```
>      <div v-if="isMobile"/>
> 
>      <div v-if="!isMobile"/>
> ```
> 
> Note how I don't use the else. I think when you have a big chunk of code in the if, it's more readable to not have the else.
>